python/manylinux1/scripts/build_virtualenvs.sh

```diff
@@ -28,7 +28,7 @@ for PYTHON in ${PYTHON_VERSIONS}; do
     PATH="$PATH:$(cpython_path $PYTHON)"
 
     echo "=== (${PYTHON}) Installing build dependencies ==="
-    $PIPI_IO "numpy==1.9.0"
+    $PIPI_IO "numpy>=1.10.0"
```

Comment:
Please pin exactly here, we need the most minimal NumPy version at build time to stay compatible with as many numpy versions as possible.